### PR TITLE
Upgrade demo apps to NativeScript 6.0.0

### LINKED ISSUE #49: `demo-ng` throws an non critical error in console when started
When you start the demo-ng app on iOS the following error is thrown:

> Successfully synced application com.tangrainc.imageswipedemong on device BA5F8C17-1C0D-41EF-8DA6-D7AF6503F817.
> Timestamp                       (process)[PID]
> CONSOLE INFO file:///node_modules/nativescript-dev-webpack/hot.js:3:0 HMR: Hot Module Replacement Enabled. Waiting for signal.
> CONSOLE LOG file:///node_modules/@angular/core/fesm5/core.js:23837:0 Angular is running in the development mode. Call enableProdMode() to enable the production mode.
> CONSOLE ERROR [native code]: ERROR TypeError: undefined is not an object (evaluating 'scrollView.contentSize = CGSizeMake(this.items.length * calculatedWidth, height)')
> CONSOLE ERROR [native code]: ERROR CONTEXT {
> "view": {
> "def": {
> "nodeFlags": 33603585,
> "rootNodeFlags": 33554433,
> "nodeMatchedQueries": 0,
> "flags": 0,
> "nodes": [
> {
> "nodeIndex": 0,
> "parent": null,
> "renderParent": null,
> "bindingIndex": 0,
> "outputIndex": 0,
> "checkIndex": 0,
> "flags": 33554433,
> "childFlags": 49152,
> "directChildFlags": 49152,
> "childMatchedQueries": 0,
> "matchedQueries": {},
> "matchedQueryIds": 0,
> "references": {},
> "ngContentIndex": null,
> "childCount": 1,
> "bindings": [],
> "bindingFlags": 0,
> "outputs": [],
> "element": {
> "ns": "",
> "name": "ActionBar",
> "attrs": [
> [
> "",
> "class",
> "action-bar"
> ],
> [
> "",
> "title",
> <…>

### PR DESCRIPTION
**Pull request by VladimirAmiorkov**
I notice that my previous PR was merged with failing travis build due to demo apps not migrated to NativeScript 6.0.0. While you deleted the travis build and merge and released my PR I think for the future of the plugin it is best to not completely remove the travis stage that is executed in new PRs because this can and will cause the plugin to not work in the future. I would like to hear your opinion on what should be expected from a PR to be considered "appropriate" for merge and release because merging without any kind of testing is sure to lead to issues down the road.

For now here is what this PR brings:

- chore: migrate to using 'nativeViewProtected' over ios/android
- chore: migrate demo-ng to NativeScript 6.0.0
- chore: migrate demo to NativeScript 6.0.0
- chore: show demos in VS Code
- chore: remove note about not supported 6.0.0
- chore: return travis config
- fix: resolves https://github.com/PeterStaev/nativescript-image-swipe/issues/49

### CODE REVIEW

File: image-swipe-common.ts [hunk 1]
```diff
@@ -89,8 +89,6 @@ export const itemsProperty = new Property<ImageSwipeBase, any[] | ItemsSource>({
         if (newValue instanceof ObservableArray) {
             addWeakEventListener(newValue, ObservableArray.changeEvent, target.refresh, target);
         }
-
-        target.refresh();
```
Comment (PeterStaev) [inline image-swipe-common.ts]: Dont think removing this is correct. Calling `refresh()` from my previous testing is obligatory and needed in order on change of the items the views to be updated correctly. Even the core `list-view` plugins has this: https://github.com/NativeScript/NativeScript/blob/5f22594eb78f4950943766f280901e38ae9de5ab/tns-core-modules/ui/list-view/list-view-common.ts#L181 

In order to handle the error that this is supposed to fix, it is better to check for iOS if the view is init here:
https://github.com/PeterStaev/nativescript-image-swipe/blob/3f0336520699f6ad03e3c69e6c0578868b200742/image-swipe.ios.ts#L302

There is already a similar check for android here:
https://github.com/PeterStaev/nativescript-image-swipe/blob/3f0336520699f6ad03e3c69e6c0578868b200742/image-swipe.android.ts#L78-L80

**Review by PeterStaev — CHANGES_REQUESTED** [review]

**Comment by PeterStaev** [comment]
Hey @VladimirAmiorkov , thank you for this update! Now onto the topic for demo apps being included in the CI/CD. This is a bit of trade off - it is true that having these ensure at least some basic guarantee that the plugin would be working, but also contributors MUST update the demos in order for me to be able to publish the new version on NPM. And I have rarely seen people actually doing this (you are the first one in quite a while 😉 ). If they don't update demos, the PRs might be hanging around for a while, just because there was some breaking change in the NS app workflow and not because the plugin is not working. 

For now, let's say that I am OK'ish with returning those and we will see how it will playout in the future.

File: image-swipe-common.ts (hunk 1 above)
Comment (VladimirAmiorkov) [reply image-swipe-common.ts]: I was initially thinking the same. But wasn't able to reproduce a scenario where this removal was an issue. I will rework this to make a check in the platform if the `nativeViewProtected` is present, this was my first solution to this issue.

**Comment by VladimirAmiorkov** [comment]
Hi @PeterStaev ,

First of all no problem, I love contributing to useful projects. Yes that is true about the demo apps and CI, there is a trade off but one of the things any contributor would want and actually has to do is test their fix/feature before making any PRs. If they don't then it is very likely their PR is not correct, of course that may not be the case for very straight forward PR changes but in the big picture any contributor that has not ran any of the apps in a repo he is contributing to is likely to have resolves partially the issue he may had and not the actual issue. This comes from personal experience with contributions to big open source projects to which I have made such mistakes due to the project overwhelming build structure etc causing me to not be able to run their "development apps".

To me personally if the demo apps in any open source project do not start then anyone that wants to work on it has to resolve and fix them before they can actually start working on the thing they wanted to fix. Is this going to stop anyone from contributing, probably yes but such (blind) contributions will do more harm than good. Making blind fixes with zero actual application testing is never a good idea and wont lead to anything good. This is my personal opinion and to me if I see an plugin with failing tests I would like to contribute to it to make those builds green :)

File: image-swipe.ios.ts [hunk 2]
```diff
@@ -300,13 +302,15 @@ export class ImageSwipe extends ImageSwipeBase {
...
     private _calcScrollViewContentSize() {
         const scrollView: UIScrollView = this.nativeViewProtected;
-        const width = utils.layout.toDeviceIndependentPixels(this.getMeasuredWidth());
-        const height = utils.layout.toDeviceIndependentPixels(this.getMeasuredHeight());
-        const safeAreaInsets = this.getSafeAreaInsets();
-        const insetAllowance = utils.layout.toDeviceIndependentPixels(safeAreaInsets.left + safeAreaInsets.right);
-        const calculatedWidth = width + insetAllowance;
-
-        scrollView.contentSize = CGSizeMake(this.items.length * calculatedWidth, height);
+        if (scrollView) {
```
Comment (PeterStaev) [inline image-swipe.ios.ts]: Let's just reverse the logic and short circuit execution with a `return` here so we don't have to surround the whole function body.

File: image-swipe.ios.ts [hunk 3]
```diff
@@ -132,27 +132,29 @@ export class ImageSwipe extends ImageSwipeBase {
 
     private _loadCurrentPage(page: number) {
         const scrollView: UIScrollView = this.nativeViewProtected;
-        const pageWidth = scrollView.frame.size.width;
+        if (scrollView) {
```
Comment (PeterStaev) [inline image-swipe.ios.ts]: Let's just reverse the logic and short circuit execution with a `return` here so we don't have to surround the whole function body.

**Review by PeterStaev — CHANGES_REQUESTED** [review]

File: image-swipe.ios.ts (hunk 2 above)
Comment (VladimirAmiorkov) [reply image-swipe.ios.ts]: I agree

File: image-swipe.ios.ts (hunk 3 above)
Comment (VladimirAmiorkov) [reply image-swipe.ios.ts]: I agree

**Pull request merged**